### PR TITLE
Support @faker js v7.2.0

### PR DESCRIPTION
**Pull request by Hourglasser23**
https://github.com/iendeavor/yup-schema-faker/issues/179

I fix that but not sure about:
- version raise (I changed major in because projects with older faker will fail)
- changelog update
- readme update 
- fixing seed function for 'yarn build' (it failed before - value needs to be in array which is required by type)

Please just give me advice how to adjust commit or adjust it yourself, so we can use your great library in our project, thanks :)

plus it is maybe more like enhancement than bugfix, so sorry for that

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Install `yup-schema-faker` and its peer dependencies with your favorite package
 ```sh
 pnpm add yup@^0.32.11
 pnpm add -D yup-schema-faker@^3.0.0
```
Comment (HonzaMac) [inline README.md:30]:
```suggestion
pnpm add -D yup-schema-faker@^4.0.0
```

**Comment by HonzaMac** [comment]
cc @iendeavor could you take a look 🙏?

File: src/random.ts [hunk 2]
```diff
@@ -8,5 +8,5 @@ export const randexp = (pattern: string | RegExp, flags?: string) => {
 }
 
 export const seed = (value: number) => {
-  Faker.seed.call(Faker, value)
+  Faker.seed.call(Faker, [value])
```
Comment (iendeavor) [inline src/random.ts:11]: @Hourglasser23 Please revert this. (Fixed in https://github.com/iendeavor/yup-schema-faker/commit/260b9f9802284185ebfe01c018138476f73f992b)

**Review by iendeavor — CHANGES_REQUESTED** [review]

**Comment by iendeavor** [comment]
@Hourglasser23 Please take a look at [this](https://github.com/Hourglasser23/yup-schema-faker/pull/1) 🙏.

File: src/random.ts (hunk 2 above)
Comment (Hourglasser23) [reply src/random.ts:11]: Supper you changed your mind, no reverting then :)

**Comment by Hourglasser23** [comment]
> @Hourglasser23 Please take a look at [this](https://github.com/Hourglasser23/yup-schema-faker/pull/1) 🙏.



> 

Np, tomorrow I'll check that :)

**Comment by Hourglasser23** [comment]
@iendeavor Should be ready 🙏🏻

**Review by iendeavor — APPROVED** [review]
Thank you 👍.

**Pull request merged**